scripts/jenkins/qa_devstack.sh

```diff
@@ -107,14 +113,28 @@ function h_setup_extra_disk {
 }
 
 function h_setup_devstack {
-    $zypper in git-core which ca-certificates-mozilla net-tools git-review
-    $zypper in 'group(nogroup)'
+    $zypper in git-review
+    $zypper mr -d SUSE_Factory_Head
+    $zypper in git-core which ca-certificates-mozilla net-tools
+    if ! getent group nobody >/dev/null; then
+        $zypper in 'group(nogroup)'
```

Comment:
This was a bug in dnsmasq which is already fixed. Did you try with updates enabled?

Reply:
Ah, I was wondering why you added that line - unfortunately the commit message didn't explain why.  I added the `if` condition around it to avoid a failure on SP3 since there `filesystem` provides the `nogroup` group but not the `group(nogroup)` virtual rpm package.

`qa_devstack.sh` already adds the updates repo (see lines 83 and 86 above) but it doesn't do a `zypper up` afterwards - maybe I should add that.

Reply:
ah, ok, it was never fixed for SP3. is there a particular reason you're trying to enable SP3 now that SP4 is out? I"m actually in progress of removing C:O:M building for SP3.

Reply:
No particular reason, I just thought it might be useful.  But regardless I think it still makes sense to only try to install `group(nogroup)` if the `nobody` group is missing.